myhpi/core/templates/core/minutes.html

```diff
@@ -33,18 +29,14 @@ <h1 class="side-panel-title">{% translate "Minutes taker" %}</h1>
                 <aside class="side-panel border-accent">
                     <h1 class="side-panel-title">{% translate "Participants" %}</h1>
                     <ul>
-                        {% for participant in page.participants.all %}
-                            <li>{{ participant.get_full_name }}</li>
-                        {% endfor %}
+                        {% for participant in page.participants.all %}<li>{{ participant.get_full_name }}</li>{% endfor %}
```

Comment:
I think having the loop body in the same line makes it much more harder to read it. Can we change that behaviour?

Reply:
No ideal solution here, I am afraid. We have two options:

1. Reduce `max_line_length` to a smaller value, like `80` (current default `120`). This would break long (i.e. difficult to read) blocks earlier, making your example easier to read. But there are still shorter (i.e. easier to read) `for` blocks that would be in one line.
2. Set `line_break_after_multiline_tag=true`. While this consistently breaks all `for` blocks, it also breaks *every* other tag that *could* contain content over multiple lines. Example: `<i>{% translate "No guests" %}</i>` would be formatted to

```
<i>
    {% translate "No guests" %}
</i>
```

Those unnecessary breaks would happen *a lot*.

That's why I would go with 1. You can see the changes to the code base in the last commit

Reply:
Sad that there is no better option :(

The shorter line length looks okay, although this also produces _some_ instances of the example you gave in 2. This made me think whether we should prefer 2. for a more consistent approach. But I'd also be fine with 1.

@frcroth any thoughts?

Reply:
It could, but only if the line is already very long/complicated. Plus the outer tags usually have more attributes if the line is long.

So, I think it's still reasonable to go for 1, since in practice it keeps the better balance most of the time

Reply:
I am more in favor of 1 but no strong opinion